homeassistant/components/ecobee/manifest.json

```
@@ -3,6 +3,7 @@
   "name": "ecobee",
   "codeowners": [],
   "config_flow": true,
+  "dependencies": ["http", "repairs"],
```

We don't have to depend on repairs anymore.

```suggestion
  "dependencies": ["http"],
```